pyxpp/parser/parser.py

```diff
@@ -6,7 +6,14 @@
 
 class CodeParser:
     def __init__(self, input_file: Path):
-        pass
+        self.input_file = input_file
 
     def consistently(self):
-        pass
+        """Returns generator object which contains character sequence of file.
+           At the end of file returns 'EOF' string"""
+        file_content = self.input_file.read_text()
```

Comment:
The meaning of the generator is lost.
`file_content` now has a huge weight in memory, since it is a string.

use `input_file.open ("r")`